app/test_split_integer.py

```diff
@@ -2,20 +2,27 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(20, 6)) == 20
+    assert sum(split_integer(8, 2)) == 8,\
+        "Sum of the parts should be equal to start integer value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
```

Comment:
You can simplify the test further  by checking that the returned list is as expected.
assert split_integer(4, 2)  == [ 2, 2 ]